src/commands/admin/resetlevels.ts

```diff
@@ -0,0 +1,43 @@
+import { DB, FIRST_LEVEL, ROLES } from '@root/config';
+import { botMasterPerms } from '@root/src/lib/permissions';
+import { Command } from '@root/src/lib/types/Command';
```

Comment:
```suggestion
import { botMasterPerms } from '@lib/permissions';
import { Command } from '@lib/types/Command';
```
Make full use of those aliases